src/NHibernate.Test/NHSpecificTest/GH3334/Entity.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections.Generic;
+
+namespace NHibernate.Test.NHSpecificTest.GH3334
+{
+	public class Entity
+	{
+		public virtual int Id { get; set; }
+ 		public virtual string Name { get; set; }
+        public virtual ISet<ChildEntity> Children { get; set; } = new HashSet<ChildEntity>();
+        public virtual OtherEntity OtherEntity { get; set; }
+	}
```

Comment:
Mixed tab and spaces.
```suggestion
	{
		public virtual int Id { get; set; }
 		public virtual string Name { get; set; }
		public virtual ISet<ChildEntity> Children { get; set; } = new HashSet<ChildEntity>();
		public virtual OtherEntity OtherEntity { get; set; }
	}
```